SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,10 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||vlive.tv/globalv-web/vam-web/old/vod/*/play
+||apis.naver.com/videostat/videostat/PlayTime.json
+||apis.naver.com/rmcnmv/rmcnmv/PlayCount.json
```

Comment:
> ||vlive.tv/globalv-web/vam-web/old/vod/*/play
||apis.naver.com/videostat/videostat/PlayTime.json
||apis.naver.com/rmcnmv/rmcnmv/PlayCount.json

Not sure about these. Better do not block them.

Reply:
Yep, I'm not sure about them too.

Reply:
OK. I will change them.
Please wait.

Reply:
Updated at 18025a6b428c0898ad8ca52dcdae50fc12ad5dcb

Reply:
I guess it's ok now. One tip for future - if you add some changes, don't put them at the beggining.